Addded hover for Blogs , Listing ,Service -cards also gave icons to profile,language,services,emergency,contact us.

dodithakur opened the pull request:
Implemented consistent hover effects with light blue highlight for:

Services cards (Diagnose, Medicines, Vaccination, Appointment)

Doctors listing cards (Oncology, Pediatrics, Orthopedics, etc.)

About section buttons (Vision, Mission, Strategy)

Blog cards (latest articles)

Added Font Awesome icons to About section buttons

Ensured transitions are smooth using transition: 0.3s ease

No changes to the main branch; all work is in feature-branch

adiprem73 commented:
Hey @dodithakur 
Can you please attach screenshots  of the things you changed or edited??... 

Thank you for you contribution

dodithakur commented:
Added icons to header 
<img width="1920" height="1020" alt="Screenshot 2025-10-05 134848" src="https://github.com/user-attachments/assets/935159cb-a804-476b-922c-be81c41a6808" />

Applied hover and color change
<img width="1920" height="1020" alt="Screenshot 2025-10-05 134943" src="https://github.com/user-attachments/assets/0d96da6a-87e7-475c-869e-887ec87ad14d" />

Added icons to vision , mission,strategy also colour change to mission and strategy when we hover
<img width="1920" height="1020" alt="Screenshot 2025-10-05 135057" src="https://github.com/user-attachments/assets/e365fb64-f999-47eb-a797-87f8fb6b3210" />

Added colour change and transition 
<img width="1920" height="1020" alt="Screenshot 2025-10-05 135313" src="https://github.com/user-attachments/assets/5d6d457d-a66b-411b-ba54-23759991b45a" />

When we hover transition occur and the article comes a bit up
<img width="1920" height="510" alt="when we hover the article get slight transition and " src="https://github.com/user-attachments/assets/3fdfc229-3bb7-4488-b3f4-059e454d0ab9" />

dodithakur commented:
I am unable to find your project under hacktoberfest projects in discord any help regarding it
